src/lib.rs

```diff
@@ -126,11 +46,20 @@ impl Collector {
     ///
     /// ```
     /// # use metrics_process::Collector;
-    /// let collector = Collector::default().prefix("my_prefix_");
+    /// let collector = Collector::new("my_prefix_");
     /// ```
-    pub fn prefix(mut self, prefix: impl Into<String>) -> Self {
-        self.prefix = prefix.into();
-        self
+    pub fn new(prefix: impl AsRef<str>) -> Self {
```

Comment:
Breaking changes without strong benefit is not acceptable.

Reply:
I'll bring back `prefix` so that it will work as before.

Reply:
Sad to use `impl Into<String>` in it though, as it is not required to build a `String`.

Reply:
Changing from `impl Into<String>` to `impl AsRef<str>` is OK because it's easy to notice and fix.